static/js/components/PersonalTab_test.js

```diff
@@ -25,6 +33,13 @@ describe("PersonalTab", () => {
     );
   };
 
+  beforeEach(() => {
+    if ( window.localStorage === undefined ) {
+      window.localStorage = localStorageMock();
+    }
+    store = configureMainTestStore(rootReducer);
```

Comment:
You could use an `integrationTestHelper` here to get this for free - might be worth it even if you're not going to use it's various test methods (`listenForActions` and so on).